### PR TITLE
fix: add unit tests for formatBalance()

### PR DESCRIPTION
**Pull request by UncleBill**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -59,13 +59,15 @@
...
+    "jest": "^26.6.3",
     "lint-staged": "^10.5.3",
     "npm-run-all": "^4.1.5",
     "only-allow": "^1.0.0",
     "prettier": "^2.1.2",
     "ts-node": "^9.1.1",
     "ttypescript": "^1.5.12",
-    "typescript": "4.1.3"
+    "typescript": "4.1.3",
+    "webcrypto": "^0.1.1"
```
Comment (septs) [inline package.json:70]: why add to workspace root?

Comment (UncleBill) [reply package.json:70]: It's required by https://github.com/DimensionDev/Maskbook/blob/5aa9aeade85b2dc4198075467010f3654e622a47/scripts/jest-setup.js#L1-L14

File: packages/maskbook/src/plugins/Wallet/formatter.test.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        t: (str: string) => str,
+    },
+}))
+import { formatBalance } from './formatter'
+
+describe('formatter', () => {
+    describe('formatBalance', () => {
+        it.each<[rawValue: BigNumber.Value, decimals: number, significant: number, result: string]>([
+            ['0', 2, 2, '0'],
+            ['0.1', 2, 2, '0'],
```
Comment (UncleBill) [inline packages/maskbook/src/plugins/Wallet/formatter.test.ts:13]: I would appreciate for case additions from anyone.

File: package.json (hunk 1 above)
Comment (septs) [reply package.json:70]: CC @Jack-Works

File: packages/maskbook/src/plugins/Wallet/formatter.test.ts (hunk 2 above)
Comment (septs) [reply packages/maskbook/src/plugins/Wallet/formatter.test.ts:13]: CC @guanbinrui

File: package.json (hunk 1 above)
Comment (Jack-Works) [reply package.json:70]: Please move those tests into the `packages/maskbook`

Comment (UncleBill) [reply package.json:70]: They are in `packages/maskbook`.

File: packages/maskbook/src/plugins/Wallet/formatter.test.ts (hunk 2 above)
Comment (guanbinrui) [reply packages/maskbook/src/plugins/Wallet/formatter.test.ts:13]: We had unit test cases before. Now it looks they have been removed. Please make your test cases run in the CI.

**Pull request closed without merging**